snyk/models.py

```diff
@@ -587,6 +587,8 @@ class Project(DataClassJSONMixin):
     attributes: Optional[Dict[str, List[str]]] = None
     _tags: Optional[List[Any]] = field(default_factory=list)
     remediation: Optional[Dict[Any, Any]] = field(default_factory=dict)
+    owner: Optional[Dict[Any, Any]] = field(default_factory=dict)
```

Comment:
Where are you getting owner from?

Upon testing this, I don't see an attribute coming back from the API for projects named "owner". (see: https://snyk.docs.apiary.io/#reference/projects/individual-project/retrieve-a-single-project)